pkg/engine/lifecycletest/pulumi_test.go

```diff
@@ -5476,3 +5476,72 @@ func TestConstructCallDependencyDedeuplication(t *testing.T) {
 		test(t, deploytest.WithoutGrpc)
 	})
 }
+
+func TestContinueOnError(t *testing.T) {
+	t.Parallel()
+	loaders := []*deploytest.ProviderLoader{
+		deploytest.NewProviderLoader("pkgA", semver.MustParse("1.0.0"), func() (plugin.Provider, error) {
+			return &deploytest.Provider{}, nil
+		}, deploytest.WithoutGrpc),
+		deploytest.NewProviderLoader("pkgB", semver.MustParse("1.0.0"), func() (plugin.Provider, error) {
+			return &deploytest.Provider{
+				DeleteF: func(urn resource.URN, id resource.ID, oldInputs, oldOutputs resource.PropertyMap,
+					timeout float64,
+				) (resource.Status, error) {
+					return resource.StatusOK, errors.New("intentionally failed delete")
+				},
+			}, nil
+		}, deploytest.WithoutGrpc),
+	}
+
+	createResource := true
+
+	programF := deploytest.NewLanguageRuntimeF(func(_ plugin.RunInfo, monitor *deploytest.ResourceMonitor) error {
+		if createResource {
+			unrelated1, _, _, _, err := monitor.RegisterResource("pkgA:m:typA", "unrelated1", true, deploytest.ResourceOptions{})
+			assert.NoError(t, err)
+			_, _, _, _, err = monitor.RegisterResource("pkgA:m:typA", "unrelated2", true, deploytest.ResourceOptions{
+				Dependencies: []resource.URN{unrelated1},
+			})
+			assert.NoError(t, err)
+
+			dependency, _, _, _, err := monitor.RegisterResource("pkgA:m:typA", "dependency", true, deploytest.ResourceOptions{})
+			assert.NoError(t, err)
+
+			_, _, _, _, err = monitor.RegisterResource("pkgB:m:typB", "failing", true, deploytest.ResourceOptions{
+				Dependencies: []resource.URN{dependency},
+			})
+			assert.NoError(t, err)
```

Comment:
We should register another resource after this one. Otherwise it might just be "failing" is the last resource to be deleted and we're not actually testing that we're continuing.

Reply:
I think with the way we compute the antichains currently this test will always fail without continuing on error.  However it doesn't hurt to add register another resource after this, to future proof the test, so I'll add it.